test_markbook.py

```diff
@@ -91,3 +91,69 @@ def test_edit_student():
     assert student["first_name"] == "Frank"
     assert student["last_name"] == "Bell"
     assert student["grade"] == 10
+
+    
+def test_create_student():
+    student = markbook.create_student("Alexander", "Jorge", "Male", None, 123456, 12, "Alexander.Jorge21@ycdsbk12.ca", [101,99, 97, 102], "Exceptional student")
+    assert student["first_name"] == "Alexander"
+    assert student["last_name"] == "Jorge"
+    assert student["gender"] == "Male"
+    assert student["image"] is None 
+    assert student["student_number"] == 123456
+    assert student["grade"] == 12
+    assert student["email"] == "Alexander.Jorge21@ycdsbk12.ca"
+    assert student["marks"] == [101, 99, 97, 102]
+    assert student["comments"] == "Exceptional student"   
+    
+    
+def test_add_assignment():
+    assignment = markbook.create_assignment("Math CPT", "October 21, 1999", 100)
+    classroom = markbook.create_classroom("MHF4U", "Advanced Functions", 3, "Mr.Smith", ["Emma", "Ethan", "Nicholas"], [])
+    markbook.add_assignment(assignment, classroom)
+    assert len(classroom["assignment_list"]) == 1
+    assert type(classroom["assignment_list"]) is list
+    
+def test_remove_assignment():
+    assignment = markbook.create_assignment ("Math CPT", "October 21, 1999", 100)
+    classroom = markbook.create_classroom("MHF4U", "Advanced Functions", 3, "Mr.Smith", ["Emma", "Ethan", "Nicholas"], [])
+    markbook.add_assignment(assignment, classroom)
+    assert len(classroom["assignment_list"]) == 1
+    markbook.remove_assignment(assignment, classroom)
+    assert len(classroom["assignment_list"]) == 0
+    assert type(classroom["assignment_list"]) is list
+    
+    
+def test_student_mark_for_assignment():
+    student = markbook.create_student("Kevin", "Wang", "Male", 234567, 9, "Kevin.Wang25@ycdsbk12.ca", [], "Nothing to report yet")
+    Mark_of_assignment = 89
+    markbook.student_mark_for_assignment(student, Mark_of_assignment)
+    assert student["marks"] == 1
+    assert student["marks"] is list
+    
+    
+def test_sort_students_alphabetically():
+    classroom = markbook.create_classroom("MHF4U", "Advanced Functions", 3, "Mr.Smith", ["Jayden","Samuel, ""Emma", "Ethan", "Nicholas"], [])
+    markbook.sort_students_alphabetically(classroom)
+    assert classroom["student_list"] == ["Emma", "Ethan", "Jayden", "Nicholas", "Samuel"]
+    assert type(classroom["student_list"]) is list  
+    
+    
+def test_order_marks():
```

Comment:
Why do you want to order the marks? How do you know to which assignment each mark belongs?